specifications/api/swagger.yaml

```
@@ -3992,6 +3992,14 @@ x-parametersSortOrder: &parametersSortOrder
       - descending
     default: descending
 
+x-parametersMaxCount: &parametersMaxCount
+  in: query
+  name: max_count
+  description: |
+    An optional maximum count.
+  schema:
+    type: integer
```

Suggestion: specify a minimum, either `0` or `1`. (Though not sure if there's much value in being able to specify `0`.)

```suggestion
    type: integer
    minimum: 1
```